GOVERNANCE.md

```diff
@@ -0,0 +1,116 @@
+# Steering Committee Governance
```

Comment:
Not saying we _need_ this, but might be interesting to explore the idea.

Reply:
I think we might omit this governance doc seeing as this template will be for mdn repos only, we can have this info on the [MDN community / about pages](https://developer.mozilla.org/en-US/docs/MDN)